README.md

```diff
@@ -1,33 +1,33 @@
 # CSK_Module_RecipeManager
-## INFO: Draft version. Not further developed / tested so far! Check for available GitHub forks of this repository to see latest updates.
 
 Module to load specific parameters for other modules.  
 
-*If available, please also add a screenshot/gif of the UI of the module here placed within /docu/media/ (see code)*
 ![](./docu/media/UI_Screenshot.png)
 
 ## How to Run
 
-[***...please fill with informations...***]  
 For further information check out the [documentation](https://raw.githack.com/SICKAppSpaceCodingStarterKit/CSK_Module_RecipeManager/main/docu/CSK_Module_RecipeManager.html) in the folder "docu".
 
 ## Information
 
-Tested on:
-|Device|Firmware|Module version
-|--|--|--|
-|e.g. SICK AppEngine|V1.5.0|V0.1.0|
-
-[***optionally***]
-Following CSK modules are used for this application via Git subtrees and should NOT be further developed within this repository (see [contribution guideline](https://github.com/SICKAppSpaceCodingStarterKit/.github/blob/main/Contribution_Guideline.md) of this GitHub organization):  
+Tested on  
 
-  * CSK_Module_XYZ (release/tag v1.2.3)
+|Device|Firmware|Module version|
+|--|--|--|
+|SICK AppEngine|v1.7.0|v1.0.0|
+|SIM1012|v2.4.2|v1.0.0|
 
 This module is part of the SICK AppSpace Coding Starter Kit developing approach.  
 It is programmed in an object oriented way. Some of the modules use kind of "classes" in Lua to make it possible to reuse code / classes in other projects.  
 In general it is not neccessary to code this way, but the architecture of this app can serve as a sample to be used especially for bigger projects and to make it easier to share code.  
 Please check the [documentation](https://github.com/SICKAppSpaceCodingStarterKit/.github/blob/main/docu/SICKAppSpaceCodingStarterKit_Documentation.md) of CSK for further information.  
 
+### Dependencies
+
+|Module|Version|Remark
```

Comment:
Also this line should have a "|" at the end to have a consistent framing of the table.

Reply:
Done